config-overrides.js

```diff
@@ -0,0 +1,23 @@
+const webpack = require('webpack');
+
+module.exports = function override(config) {
+    const fallback = config.resolve.fallback || {};
+    Object.assign(fallback, {
+        "crypto": require.resolve("crypto-browserify"),
+        "stream": require.resolve("stream-browserify"),
+        "assert": require.resolve("assert"),
+        "http": require.resolve("stream-http"),
+        "https": require.resolve("https-browserify"),
+        "os": require.resolve("os-browserify"),
+        "url": require.resolve("url")
+    })
+    config.resolve.fallback = fallback;
+    config.plugins = (config.plugins || []).concat([
+        new webpack.ProvidePlugin({
+            process: 'process/browser',
+            Buffer: ['buffer', 'Buffer']
+        })
+    ])
+    config.ignoreWarnings = [/Failed to parse source map/];
+    return config;
+}
```

Comment:
I definitely want to dig into this and see what all is required here. I imagine most of it is because we need the polyfills for those, just a headache to need to pack it all like this. Want to make sure nothing here is shaky.